actors/ppo.py

```diff
@@ -0,0 +1,720 @@
+# Copyright 2022 AI Redefined Inc. <dev+cogment@ai-r.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from dataclasses import dataclass
+from typing import Union, Tuple, List
+
+import cogment
+import numpy as np
+import torch
+from torch.distributions.normal import Normal
+
+from cogment_verse import Model
+from cogment_verse.run.run_session import RunSession
+from cogment_verse.run.sample_producer_worker import SampleProducerSession
+from cogment_verse.specs import (
+    PLAYER_ACTOR_CLASS,
+    AgentConfig,
+    EnvironmentConfig,
+    EnvironmentSpecs,
+    PlayerAction,
+    cog_settings,
+    flatten,
+    flattened_dimensions,
+    unflatten,
+)
+
```

Comment:
We might need "torch.multiprocessing.set_sharing_strategy("file_system")" for Ubuntu

Reply:
I added this command line to `ppo.py`. Please test it out if it works